.travis.yml

```diff
@@ -19,7 +28,6 @@ cache:
   directories:
   - $HOME/.composer/cache
 
-env:
```

Comment:
I think this `env:` line shouldn't be deleted, but the two `env` sections, should be merged together with the new one above.

Reply:
Yes, sure. I've merged them. Thanks!